docs/architecture/adr/0000-template.md

```
@@ -0,0 +1,63 @@
+status: proposed
+last-reviewed: YYYY-MM-DD
+owner: "@github-handle"
+applies-to: next/v1
+supersedes: []
+superseded-by: null
+compliance-impact: []
+license-impact: none
+threat-mitigation-link: null
+---
+
+# ADR-NNNN: <title>
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)